runtime-modules/bureaucracy/src/tests/mod.rs

```diff
@@ -1529,3 +1528,25 @@ fn slash_worker_stake_fails_with_not_set_lead() {
         slash_stake_fixture.call_and_assert(Err(Error::CurrentLeadNotSet));
     });
 }
+
+#[test]
+fn get_all_worker_ids_succeeds() {
+    build_test_externalities().execute_with(|| {
+        let worker_ids = Bureaucracy1::get_all_worker_ids();
+        assert_eq!(worker_ids, Vec::new());
+
+        let worker_id1 = fill_worker_position(None, None, true);
+        let worker_id2 = fill_worker_position(None, None, false);
+
+        let mut expected_ids = vec![worker_id1, worker_id2];
+        expected_ids.sort();
+
+        let mut worker_ids = Bureaucracy1::get_all_worker_ids();
+        worker_ids.sort();
```

Comment:
I'm assuming sorting is needed here because iterating over the map in get_all_worker_ids() is not guaranteed to have same order as inserting?

Reply:
Correct. This element order is guaranteed.